app/templates/views/broadcast/_broadcast_message_history.html

```diff
@@ -0,0 +1,33 @@
+<div class="govuk-grid-column-full alert-list-item">
+  <div class="govuk-grid-column-full">
+    <h2 class="message-name">{{ broadcast_message.reference }}</h2>
+    <p class="hint">
+      {% if broadcast_message.version > 1 %}
```

Comment:
Perhaps instead of checking the magic version number here you shoud create a function if "is_edited" which returns a true/false.  I'm also not keen on these manually incremented version numbers.